lib/utilities/monitoringHandler.js

```diff
@@ -64,6 +64,36 @@ const httpResponseSizeBytes = new client.Summary({
     help: 'Cloudserver HTTP response sizes in bytes',
 });
 
+const quotaEvaluationDuration = new client.Histogram({
+    name: 's3_cloudserver_quota_evaluation_duration_seconds',
+    help: 'Duration of the quota evaluation operation',
+    labelNames: ['action', 'code', 'type'],
+    buckets: [0.0001, 0.005, 0.015, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5],
```

Comment:
`0.1ms` does not seem realistic...
Also, you mentioned the impact (in your test) was around 1ms, so i guess we should have more buckets in this range: maybe `[1ms, 2,5ms, 5ms, 10ms, 25ms, 50ms, 100ms, 200ms, 400ms, 800ms]` ?

Reply:
Then you can also change the buckets for the metric_retrieval_duration metric as the quota depends on this one and could be double the metric retrieval time ?

And then maybe change as well the buckets in scuba to match with the retrieval buckets ?

___

Also could it be possible that  the time will differ between a mongodb backend for artesca and a leveldb/metadata backend for the ring and we should have different buckets configure for each product ?

Reply:
Updated in https://github.com/scality/cloudserver/pull/5556/files#diff-3daa33d35f5375a7070d74a0b174c01ab8b982392db4025ebc49cf3690aa16bcR80

I will assess if changes are needed in Scuba, as the latency in scuba is lower (no network latency added, as compared with cloudserver)